isovar/variant_read.py

```diff
@@ -148,8 +150,13 @@ def variant_read_from_single_read_at_locus(read, ref, alt):
                     read))
             return None
 
+    if sequence[read_pos_before + 1:read_pos_after] != alt:
```

Comment:
Sorry, that's a typo: I meant **R2* doesn't agree. 

That said, I'm guessing the number of dashes in those sequences isn't supposed to indicative of the deletion length.